src/main/java/com/microsoft/sqlserver/jdbc/KeyVaultTokenCredential.java

```diff
@@ -29,6 +29,8 @@
  */
 @Immutable
 class KeyVaultTokenCredential implements TokenCredential {
+    private static final String NULL_VALUE = "R_NullValue";
```

Comment:
if you do this why not define this to be the error string? 
i.e. private static final String NULL_VALUE = SQLServerException.getErrString("R_NullValue");